src/components/spacings/stack/stack.js

```diff
@@ -5,7 +5,15 @@ import getStyles from './stack.styles';
 
 const Stack = props => (
   <div css={getStyles(props)} {...filterDataAttributes(props)}>
-    {props.children}
+    {/* We need to wrap the children into a <div> to ensure that
+    the margins defined by the spacing component are properly applied.
+    Some children might have `margin: 0` defined, which results in a
+    higher CSS specificity, causing the margings of the spacing
+    components to be ignored.
+    See https://github.com/commercetools/ui-kit/issues/542 */}
+    {React.Children.map(props.children, child => (
+      <div>{child}</div>
```

Comment:
Can we maybe additionally remove the lobotomized owls selector on the parent and just have the margin applied here? Would make CSS a bit more atomic.

Reply:
We can't, because the first child doesn't have the margin.